modules/claims_api/lib/custom_error.rb

```diff
@@ -25,37 +29,50 @@ def build_error
 
     private
 
-    def raise_backend_exception(source = @error&.class, key = 'EVSS')
-      error_details = get_error_details
-      raise ::Common::Exceptions::BackendServiceException.new(
-        key,
-        { source: },
-        @status,
-        error_details
-      )
+    def raise_backend_exception(key = 'EVSS')
```

Comment:
@FonzMP

Reply:
I'm not sure what this is asking / suggesting?